### PR TITLE
actions need to be put on cpu first before conversion to numpy array

### PR DESCRIPTION
**Pull request by AGKhalil**
This is a minor fix. But without it gpu training doesnt work

### CODE REVIEW

File: squiRL/common/agents.py [hunk 1]
```diff
@@ -67,7 +67,7 @@ def get_action(
         action_logit = net(obs)
         probs = F.softmax(action_logit, dim=-1)
         dist = torch.distributions.Categorical(probs)
-        action = dist.sample().squeeze().numpy()
+        action = dist.sample().squeeze().cpu().numpy()
```
Comment (MihaiAnca13) [inline squiRL/common/agents.py:70]: does it need detach() as well?

Comment (AGKhalil) [reply squiRL/common/agents.py:70]: no it doesnt. I tested it with 1 gpu and 2 gpus. no `detach` needed.
